packages/application-components/src/components/modal-page/README.md

```diff
@@ -14,41 +14,42 @@ Modal pages are controlled components used to render a page using a modal contai
+| `baseZIndex`     | `number`           |          | -                | `1000`  | The base `z-index` value to be applied to the overlay container, incremented by `1` according to the `level` prop.                                                                                                                                                                                          |
+| `zIndex`         | `number`           |          | -                | -       | The `z-index` value to be applied to the overlay container. This value overrides the normal `z-index` value calculated from the `baseZIndex` and `level` props. If you provide this value, you would need to take care of providing a proper `z-index` based on the stacked level.                          |
+| `showTopBar`     | `boolean`          |          | -                | `true`  | Set this to `false` to not display the top bar, which contains navigational buttons for the modal, inclusing the close button.                                                                                                                                                                              |
+| `showHeader`     | `boolean`          |          | -                | `true`  | Set this to `false` to not display the header, which contains the title, subtitle and the `headerActions` node.                                                                                                                                                                                             |
+| `headerActions`  | `node`             |          | -                | -       | This node will be rendered in the header section, on the right side of the subtitle, as long as the `showHeader` prop isn't falsy.                                                                                                                                                                          |
+| `topBarLabels`   | `object`           |          |                  | `{}`    | Use this object to pass `currentPath` and `previousPath` fields as labels to the top bar.                                                                                                                                                                                                                   |
+| `topBarTone`     | `string`           |          | [`white`,`gray`] | `white` | Set the colour of the top bar section.                                                                                                                                                                                                                                                                      |
```

Comment:
I'm not sure I would use `tone`. Why not use `color` or `backgroundColor`?

Reply:
Fine by me :) I also thought `shade`, since it's only b&w, but yeah `color` is just easier to understand.

Reply:
Good point. The MC has a history of coming up with any possible name for something like this. `shade` would be something new. We have `tone`, `color` and `theme` and I think just color for these things is fine :)